frontend/components/elements/StoryItem.vue

```diff
@@ -34,22 +34,44 @@
                     <VIcon name="arrow-down"/>
                 </BButton>
 
-                <span class="js-drag-drop ml-2" v-if="isView(['backlog', 'planning-sprint'])" title="Drag to change order">
-                    <VIcon 
-                        name="arrows" 
-                        label="Drag and drop to change order"/>
+                <span
+                    :class="{ 'js-drag-drop': sortable, 'ml-2': true }"
+                    v-if="isView(['backlog', 'planning-sprint'])"
+                    @dblclick="!sortable || startKeyboardSort()"
+                    :title="sortable ? 'Drag to change order, double click to use keyboard' : 'Sort by position ascending (default order) to be able to change order'">
```

Comment:
It is important to mention, that at the end of the sorting with arrow keys, one needs to press the enter-key. :/

Reply:
I've added a bit more text